tests/suites/caas_filesystem/import.sh

```diff
@@ -0,0 +1,52 @@
+test_import_filesystem() {
+        # Echo out to ensure nice output to the test suite.
+        echo
+
+        # Ensure a bootstrap Juju model exists.
+        model_name="import-filesystem"
+        file="${TEST_DIR}/test-${model_name}.log"
+        ensure "${model_name}" "${file}"
+
+        # Create a PersistentVolume by deploying and deleting an application.
+        echo "Create persistent volume to be imported"
+        juju deploy postgresql-k8s --channel 14/stable --trust
+        wait_for "postgresql-k8s" "$(active_idle_condition "postgresql-k8s" 0)"
```

Comment:
We can do this
`wait_for_storage "attached" '.storage["pgdata/0"]["status"].current'`

and it will make the test run faster. We just care that the storage gets attached before we remove the app.